frontend/src/components/pages/HomePage/SearchAndFilters.tsx

```diff
@@ -103,6 +76,43 @@ const SearchAndFilters = ({
   setBuilding,
   setFlagged,
 }: Props): React.ReactElement => {
+  const [userLabels, setUserLabels] = useState<UserLabel[]>();
+  const [residentLabels, setResidentLabels] = useState<ResidentLabel[]>();
+
+  const getUsers = async () => {
+    const data = await UserAPIClient.getUsers({ returnAll: true });
+    const users = data?.users;
+    if (users) {
+      const labels = users.map((user: User) => {
+        return {
+          id: user.id,
+          label: `${user.firstName} ${user.lastName}`,
+          value: user.id,
+        } as UserLabel;
+      });
+      setUserLabels(labels);
+    } else {
+      setUserLabels(undefined);
+    }
+  };
+
+  const getResidents = async () => {
+    const data = await ResidentAPIClient.getResidents({ returnAll: true });
+    const residentsData = data?.residents;
+    if (residentsData) {
+      const labels = residentsData.map((resident: Resident) => {
+        return {
+          id: resident.id,
+          label: `${resident.initial}${resident.roomNum}`,
```

Comment:
Instead of this string concatenation you should use `resident.residentId` which is constructed in this format for you